integration/init/jaeger/metadata.yaml

```
@@ -0,0 +1,3 @@
+upstream: "https://github.com/jaegertracing/jaeger-kubernetes/blob/59e7afcae7b2f3bc109695cfda2a5c116cea4391/jaeger-production-template.yml"
+args: []
```

```suggestion
args: ["--prefer-git"]
```
To help the GitHub API rate limits of people running the integration tests locally